source/seedsource/tasks/cleanup_tifs.py

```diff
@@ -0,0 +1,20 @@
+from seedsource_project import settings
```

Comment:
This should be:

```python
from django.conf import settings
```

See: https://docs.djangoproject.com/en/1.8/topics/settings/#using-settings-in-python-code